console/console.go

```diff
@@ -21,7 +21,7 @@ func Setup(rootfs, consolePath, mountLabel string) error {
 		return err
 	}
 
-	if err := os.Chown(consolePath, 0, 0); err != nil {
+	if err := os.Chown(consolePath, hostRootUid, hostRootUid); err != nil {
```

Comment:
We could make that assumption since it doesn't make sense to have separate uid/gid mappings. (Even though we have exposed that).